### PR TITLE
Cassiopeia Privacy Consent - Display Options Inline

### PR DESCRIPTION
**Pull request by coolcat-creations**
Pull Request for Issue #37645 .

### Summary of Changes
The Privacy Consent decision is placed inline. It saves space and has a better usability 

### Testing Instructions
Turn on the Privacy consent plugin 
Turn on the Terms of Service plugin
Apply Patch an see if the radio buttons on the registration page and on the profile edit page display correct

### Actual result BEFORE applying this Pull Request
grafik

### Expected result AFTER applying this Pull Request
grafik

### Documentation Changes Required
No

### CODE REVIEW

File: build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss [hunk 1]
```diff
@@ -11,6 +11,15 @@
   background: $white;
   box-shadow: -3px -2px 22px #ddd;
 
+  nav {
```
Comment (brianteeman) [inline build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss]: this file is not part of this pr please remove

Comment (coolcat-creations) [reply build/media_source/templates/site/cassiopeia/scss/blocks/_toolbar.scss]: Thanks, seems I did not set up the branches correctly

**Comment by richard67** [comment]
@coolcat-creations Is it by purpose that your PR now deletes the _toolbar.scss file instead of just reverting the unwanted changes in it?

**Comment by richard67** [comment]
@coolcat-creations I think the reason for problem with the unwanted changes in the _toolbar.scss files is that you have created the branch for this PR based on the branch for your other PR #37707 instead of your 4.1-dev branch, possibly because the branch of the other PR was actual branch, and when creating a new branch, git defaults to the actual branch as base branch if you don't specify something else.

Later you tried to fix that by deleting the file, but that's wrong. You should have replaced it by the one from a clean 4.10-dev branch.

The easiest way to fix this now is to close this PR and do it again with another, new branch, this time based on your 4.1-dev branch (or better, the 4.1-dev branch of the upstream repo = this repo here).

For the mean time I leave the issue open because this PR here is wrong like it is now and so will not fix the issue without breaking something else.

**Comment by coolcat-creations** [comment]
oh no sorry :-D I fix this

**Comment by coolcat-creations** [comment]
created https://github.com/joomla/joomla-cms/pull/37709

**Comment by richard67** [comment]
Thanks.
